endpoints/openrtb2/auction.go

```diff
@@ -852,7 +851,7 @@ func validateNativeAssetVideo(video *nativeRequests.Video, impIndex int, assetIn
 }
 
 func validateNativeAssetData(data *nativeRequests.Data, impIndex int, assetIndex int) error {
-	if data.Type < native1.DataAssetTypeSponsored || data.Type > native1.DataAssetTypeCTAText {
+	if data.Type < native1.DataAssetTypeSponsored || (data.Type > native1.DataAssetTypeCTAText && data.Type < 500) {
```

Comment:
I am not sure why we don't have the asset type check for the image asset as well but can we add that too in the `validateNativeAssetImage` method? From the spec, it looks like Image can have exchange specific types as well

Reply:
It can, yes. I also don't know why we don't validate the image asset type. I'll add that in a follow-up PR.